src/RpcServer/RpcError.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2015-2023 The Neo Project.
+//
+// The Neo.Network.RPC is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Json;
+
+namespace Neo.Plugins
+{
+    public class RpcError
+    {
+        #region Default Values
+
+        //public static readonly RpcError InternalServerError = new(-32603, "Internal server RpcError");
```

Comment:
Please, do not comment out the unused ones. We have RPC client that (in future) might use these error codes to be able to return named errors. And almost all of these commented errors supposed to be used.